oai_agents/common/tags.py

```diff
@@ -67,8 +68,9 @@ def map_to_index(teamtype):
             TeamType.SELF_PLAY: 9,
             TeamType.SELF_PLAY_LOW: 10,
             TeamType.SELF_PLAY_MEDIUM: 11,
-            TeamType.SELF_PLAY_HIGH: 12,
-            TeamType.ALL_MIX: 13
+            TeamType.SELF_PLAY_MIDDLE: 12,
```

Comment:
What is the difference between `SELF_PLAY_MIDDLE` and `SELF_PLAY_MEDIUM`

Also imo it would be better not to overwrite the enum values so old data is still interpretable. Could be a bit weird for the graph if the value of `SELF_PLAY_MIDDLE` is 14 but at least we know 14 is has always been `SELF_PLAY_MIDDLE`. Alternatively if the value doesnt need to be an int and is only used in the chart you could do something like `12.5`

Reply:
Good point, I changed it to 12.5. 

[1, 5, 8, 8, 10]
medium: closest to 10+1/2 =~ 5 --> so 5
middle: the one that's in the middle of the sorted list (index wise): 8